tests/ui/test_ui_tools.py

```diff
@@ -552,7 +553,8 @@ def test_mouse_event(self, mocker, stream_view, mouse_scroll_event, widget_size)
         row = 1
         focus = "WIDGET"
         stream_view.mouse_event(size, event, button, col, row, focus)
-        stream_view.keypress.assert_called_once_with(size, key)
+        stream_view.keypress.assert_called_with(size, key)
+        assert stream_view.keypress.call_count == SIDE_PANELS_MOUSE_SCROLL_LINES
```

Comment:
These only assert the last call is called a certain way, and that overall it's a certain number of times. You likely want `has_calls`.